reference/resources/felixconfig.md

```diff
@@ -92,6 +92,11 @@ spec:
 | vxlanMTU                           | MTU to use for the VXLAN tunnel device. Also controls NodePort MTU when eBPF enabled. | int | int | `1410` |
 | vxlanPort                          | Port to use for VXLAN traffic. A value of `0` means "use the kernel default". | int | int | `4789` |
 | vxlanVNI                           | Virtual network ID to use for VXLAN traffic. A value of `0` means "use the kernel default". | int | int | `4096` |
+| wireguardEnabled                   | Enable encryption on WireGuard supported nodes in cluster. When enabled, pod to pod traffic will be sent over encrypted tunnels between the nodes. | `true`, `false` | boolean | `false` |
+| wireguardListeningPort             | Port used by WireGuard tunnels. Felix sets up WireGuard tunnel on each node specified by this port. Please note that this can only be setup in global `FelixConfiguration` resource. Setting it per host, config-file or environment variable will not work. | int | int | 51820 |
+| wireguardRoutingRulePriority       | WireGuard routing rule priority value set up by Felix. Change this to appropriate value if the default value can have an impact on existing routing rules on the node. | int | int | 99 |
+| wireguardInterfaceName             | Name of the WireGuard interface created by Felix. If the value is changed, Felix will not delete earlier configured interface and must be manually deleted on each node, if needed. | string | string | wireguard.cali |
```

Comment:
Name of the WireGuard interface created by Felix. If you change the name, and want to clean up the previously-configured interface names on each node, this is a manual process.